src/ctap/mod.rs

```diff
@@ -763,11 +763,13 @@ impl CtapState {
         env.user_presence().check(channel)?;
         self.client_pin.clear_token_flags();
 
+        let default_cred_protect = env.customization().default_cred_protect();
+
```

Comment:
nit: I would remove this empty line. This variable is "part" of the following block which deals with cred protect.

Reply:
Done. And I want to merge these first because the next one has some more refactors (introducing TestCustomization that makes array ref constants configurable).

Reply:
Sounds good.